### ISSUE TITLE
Custom Route with multiple waypoints in turn By turn Navigation

### ISSUE THREAD

**Issue by khem88** [tallygo/TallyGoAndroid#4]
Is we can create the custom route to add a multiple waypoints with the help of tallyGo Sdk? . If yes please tell me how?

Thanks

**Comment by mtriplett** [comment]
Yes. New implementation examples, including Waypoints, are coming soon. Stay tuned.

**Comment by mtriplett** [comment]
Waypoints are planned for Q2 this year. I will update this thread when we ship it.

In the mean time, I suggest you check out v2.0 of our SDK which is a major update loaded with functionality, features, and fixes.
